### PR TITLE
Use Cases (Library Description)

### PR DESCRIPTION
**Pull request by QABoahene**
Sorry for the delay, exhausting couple of weeks for me. Could you read the introduction and let me know if it's okay?
I will also be working on the code for the `Usage`. I will update it if done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,24 @@ Evaluating machine learning models is an important aspect of building models. Th
 
 For instance, in order to map a logistic regression value to a binary category, you must define a classification threshold (also called the decision threshold). In say a cancer patient classification, a value above that threshold indicates "Patient has cancer"; a value below indicates "Patient does not have cancer." It is tempting to assume that the classification threshold should always be 0.5, but thresholds are problem-dependent, and are therefore values that you must tune.
```
Comment (mawuliadjei) [inline README.md:8]: 'logistic regression value' should be 'probability representation from logistic regression'

**Review by mawuliadjei — COMMENTED** [review]
Overall I think this is really good. I'm happy that we've reached this stage of the project where we can describe what we've built.

I have a few comments.

1. I think in the introduction, we should introduce what a binary classification is so we make it clear that this library is specifically for binary classification tasks. Especially the fact that the metrics you listed are mainly used for binary classification tasks.
2. I think we should get some diagrams to illustrate the concept and problem we're trying to solve. We can ask @tobelum  to work on the diagrams 
3. I owe you code for the other stuff.

File: README.md (hunk 1 above)
Comment (QABoahene) [reply README.md:8]: Got it.

**Comment by QABoahene** [comment]
Updated the Readme. Will be waiting for the rest.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Budgies
 This python library allows you to conveniently evaluate predicted probablilities during a binary classification task by presenting you with the optimum probability thresholds.
 
-
 ## Introduction
+*Classification* tasks in machine learning involves models or algorithms learning to *assign class lables* to elements of a set. **Binary Classification** is the process of assignig elements to two class labels on the basis of a classification rule. Some of the examples of binary classification includes classifying mails under 'spam' or 'not a spam', medical tests ('cancer detected' or 'cancer not detected') and churn prediction ('churn' or 'not').
```
Comment (mawuliadjei) [inline README.md:5]: typo in 'assigning'

**Review by mawuliadjei — APPROVED** [review]

**Comment by mawuliadjei** [comment]
Resolve the comment I gave and feel free to merge into master. I'll work on the code over the weekend.

**Pull request merged**